arbnode/transaction_streamer.go

```diff
@@ -1136,7 +1202,310 @@ func (s *TransactionStreamer) executeMessages(ctx context.Context, ignored struc
 	return s.config().ExecuteMessageLoopDelay
 }
 
+func (s *TransactionStreamer) PollSubmittedTransactionForFinality(ctx context.Context) time.Duration {
```

Comment:
Remaining changes in this file are for the sovereign sequencer mode. They handle the submission of L2 blocks to the Espresso Sequencer for notarization.